pkg/api/annotations.go

```diff
@@ -34,6 +35,16 @@ func (hs *HTTPServer) GetAnnotations(c *models.ReqContext) response.Response {
 		SignedInUser: c.SignedInUser,
 	}
 
+	// When dashboard UID present in the request, we ignore dashboard ID
```

Comment:
should we ignore the `dashboardId` or return for both?
if we decide the former then we need to mention it in the docs/spec

Reply:
Let me update the doc